tsconfig.test.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "./tsconfig.src.json"
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true
```

Comment:
You need this so the test/ folders doesn't get cluttered with .d.ts and .js and .js.map files

Reply:
Nice!